src/transformers/models/detr/feature_extraction_detr.py

```diff
@@ -400,6 +400,52 @@ def _normalize(self, image, mean, std, target=None):
 
         return image, target
 
+    def binary_mask_to_rle(self, mask):
```

Comment:
I think both of these methods `binary_mask_to_rle` and `remove_low_and_no_objects` should be moved out of the class to be top-level functions: they are helper functionalities used by the feature extractor to process.